schemas/bookbrainz.sql

```diff
@@ -610,6 +610,77 @@ CREATE TABLE bookbrainz._editor_entity_visits (
 ALTER TABLE bookbrainz._editor_entity_visits ADD FOREIGN KEY (editor_id) REFERENCES bookbrainz.editor (id);
 ALTER TABLE bookbrainz._editor_entity_visits ADD FOREIGN KEY (bbid) REFERENCES bookbrainz.entity (bbid);
 
+CREATE TABLE IF NOT EXISTS bookbrainz.import (
+    id SERIAL PRIMARY KEY,
+    type bookbrainz.entity_type NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS bookbrainz.creator_import (
+	import_id INT PRIMARY KEY,
+	data_id INT
```

Comment:
I think data should be "NOT NULL" here (and for the other `<type>_import` tables). I don't expect an import object to exist without associated data. Not entirely sure that the revision should be nullable for entity_header table either (which I believe this is based on?), but will need top review that.

Reply:
Definitely data_id must not be null. Will make relevant changes.
The *entityType_import* table is basically coalesced version of *entityType_header* and *entityType_revision* tables. There is no *revision* related field here, so not sure what is being referenced here. :|
The plan is to save all the data as it is the schema changes in the PR, then use create entity page and site side callback to create a new entity.

Reply:
Yeah, I was talking about the revision_id on entity_header - I thought you might have made data_id nullable here because revision_id is nullable there.